server/core/store.go

```diff
@@ -193,8 +194,8 @@ func (s *StoreInfo) IsDisconnected() bool {
 // GetLabelValue returns a label's value (if exists).
 func (s *StoreInfo) GetLabelValue(key string) string {
 	for _, label := range s.GetLabels() {
-		if label.GetKey() == key {
-			return label.GetValue()
+		if strings.EqualFold(label.GetKey(), key) {
+			return strings.ToLower(label.GetValue())
```

Comment:
I prefer returning `label.GetValue()` here, use `strings.EqualFold` outside to compare.